std/machines/mod.asm

```diff
@@ -1,9 +1,12 @@
 mod arith;
+mod arith_bb;
+mod arith_bb_mul;
 mod binary;
 mod binary_bb;
 mod range;
 mod hash;
 mod memory;
+mod memory_bb;
```

Comment:
What about, `arith16`, `memory16`, `arith16_mul` as we already name is shift16 by suggestion?

Reply:
yes we'll try to keep 16 vs 32 where the constraints are based on the size of the field, and bb, m31, gl, where the field is relevant, like in poseidon